ArmPose arm_pose
geometry_msgs/Pose target
float32 gripper_offset
float32 eff_offset
---
Trajectory[] trajectories